src/main/kotlin/me/apella/chat/util/FileUtils.kt

```
@@ -0,0 +1,22 @@
+
+@Slf4j
+class FileUtils {
+    fun readFileFromLocation(fileUrl: String): ByteArray? {
+        if (fileUrl.isBlank()) {
+            return byteArrayOf(0)
+        }
+
+        return try {
+            val filePath = File(fileUrl).toPath()
+            Files.readAllBytes(filePath)
+        } catch (e: IOException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)